### DISCUSSION TITLE
Min Width Headers on list view web part

### DISCUSSION THREAD

**Discussion by mmmiller73** [pnp/sp-dev-fx-webparts#1807] [General]
I am new to spfx but not to SharePoint.  Has anyone been able to get min width to work for headers for all users in a list/library view web part?  I need to show all documents from a view and need the ability to filter a Category field but have the requirement for the headers to stay a certain width or at least not be smaller than a certain width for ALL users.  
[minWidthHeaders.docx](https://github.com/pnp/sp-dev-fx-webparts/files/6252999/minWidthHeaders.docx)

**Comment by DonKirkham** [comment]
I'm not sure I understand your question, but if you resize the columns in a view, then save the view (overwrite the old view), the column widths should stick.
